IPython/core/oinspect.py

```diff
@@ -702,15 +702,22 @@ def pinfo(
             del info['text/html']
         page.page(info)
 
-    def info(self, obj, oname="", formatter=None, info=None, detail_level=0):
-        """DEPRECATED since 5.0. Compute a dict with detailed information about an object."""
-        if formatter is not None:
-            warnings.warn('The `formatter` keyword argument to `Inspector.info`'
-                     'is deprecated as of IPython 5.0 and will have no effects.',
-                      DeprecationWarning, stacklevel=2)
-        return self._info(obj, oname=oname, info=info, detail_level=detail_level)
+    def _info(self, obj, oname="", info=None, detail_level=0):
+        """
+        Inspector.info() was likely unproperly marked as deprecated
+        while only a parameter was deprecated. We "un-deprecate" it.
+        """
+
+        warnings.warn(
+            "The `Inspector.info()` has been un-deprecated as of 8.0 and the"
+            " `formatter=` keyword removed. `Inspector._info` is now an alias."
+            " You can always call `.info()` directly.",
```

Comment:
```suggestion
            "The `Inspector.info()` method has been un-deprecated as of 8.0 "
            "and the `formatter=` keyword removed. `Inspector._info` is now "
            "an alias, and you can just call `.info()` directly.",
```

Missing a word for grammar, and did a bit of sentence simplification.